brainstormforce/ultimate-addons-for-gutenberg
# Fix: Blockquote - Tweet link showing a empty page link and username.

### Description

- Trello - https://trello.com/c/MmedZ8yU/311-blockquote-issue-twitter-link-is-showing-a-broken-page-via

### Screenshots

- issue - https://share.bsf.io/7KuPNO0d

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop blockquote add a user name and check by clicking tweet btn is everything displaying properly in tweet window.
- Add custom URL and check the same.

### Checklist:
